src/Settings/components/comon/bottomSheetComponent.jsx

```diff
@@ -14,35 +17,86 @@ const BottomSheetComponent = ({
+  const nightStyles = nightModeColor(isNightMode);
+  const { width, height } = Dimensions.get("window");
+
+  const [orientation, setOrientation] = useState(width < height ? "PORTRAIT" : "LANDSCAPE");
+  useEffect(() => {
+    Dimensions.addEventListener("change", ({ window: { widthDimension, heightDimenstion } }) => {
+      if (widthDimension < heightDimenstion) {
+        setOrientation(constant.PORTRAIT);
+      } else {
+        setOrientation(constant.LANDSCAPE);
+      }
+    });
+  }, []);
+  const bottomStyle = [];
+  if (Platform.OS === "ios") {
+    bottomStyle.push(styles.viewWrapper);
+    bottomStyle.push(orientation === constant.LANDSCAPE ? styles.width_90 : styles.width_100);
+  } else {
+    bottomStyle.push(styles.androidViewWrapper);
+  }
+
   return (
-    <BottomSheet
-      modalProps={{
-        supportedOrientations: [
-          "portrait",
-          "portrait-upside-down",
-          "landscape-left",
-          "landscape-right",
-        ],
-      }}
-      isVisible={isVisible}
-      onBackdropPress={() => toggleVisible(false)}
-    >
-      <View style={styles.viewWrapper}>
-        <Text style={[styles.bottomSheetTitle, textNightStyle, containerNightStyles]}>{title}</Text>
-        <Divider />
-        {actionConstant.map((item) => (
-          <RenderBottomSheetItem
-            key={item.key}
-            item={item}
-            toggleVisible={toggleVisible}
-            value={value}
-            action={action}
-          />
-        ))}
-      </View>
-    </BottomSheet>
+    <SafeAreaProvider>
+      <SafeAreaView>
+        <Modal
+          visible={isVisible}
+          animationType="fade"
+          transparent
+          supportedOrientations={[
+            "landscape",
+            "landscape-left",
+            "landscape-right",
+            "portrait",
+            "portrait-upside-down",
+          ]}
+        >
+          <Pressable style={StyleSheet.absoluteFill} onPress={() => toggleVisible(false)}>
+            <BlurView
+              reducedTransparencyFallbackColor="white"
+              style={styles.blurViewStyle}
+              blurType="dark"
+              blurAmount={10}
+            >
+              <View style={bottomStyle}>
+                <Text style={[styles.bottomSheetTitle, textNightStyle, containerNightStyles]}>
+                  {title}
+                </Text>
+                <Divider />
+                {actionConstant.map((item) => (
+                  <ListItem
+                    key={item.key}
+                    bottomDivider
+                    containerStyle={containerNightStyles}
+                    onPress={() => {
+                      toggleVisible(false);
+                      dispatch(action(item.key));
+                      if (item.key.includes(".mp3")) {
+                        const soundTitle = item.key.split(".mp3")[0];
+                        SoundPlayer.playSoundFile(soundTitle, ".mp3");
+                      }
```

Comment:
@amitojsingh veerji, what would be the reason for first splitting from mp3 and then adding it again in the next step. Why can't we directly pass the filename in `playSoundFile` function.


p.s. Do not resolve this comment after replying. It gets missed once its resolved.

Reply:
Veer ji it's syntax for SoundPlayer Library. Here's the reference link: [react-native-sound-player](https://www.npmjs.com/package/react-native-sound-player)

`SoundPlayer.playSoundFile(FileName, FileType);`